test/integration/default/build/route_table.tf

```diff
@@ -0,0 +1,48 @@
+#===========================================================================#
+#                                 Route Tables
+#===========================================================================#
+
+
+data "aws_internet_gateway" "default" {
+  filter {
+    name = "attachment.vpc-id"
+    values = ["${data.aws_vpc.default.id}"]
+  }
+}
+
+resource "aws_route_table" "rtb" {
+  vpc_id = "${data.aws_vpc.default.id}"
+
+  route {
+    cidr_block = "172.32.1.0/24"
+    gateway_id = "${data.aws_internet_gateway.default.id}"
+  }
+
+  tags {
```

Comment:
Minor thing, but since we're not testing for tags yet, and we have a dedicated test AWS account, I generally omit Tags.  The idea being that the things in the test fixture file are the minimal number of things we need to test, and each thing is important.

Reply:
Ok I can remove this for now.